src/apps/vpn/ui/screens/settings/ViewDNSSettings.qml

```diff
@@ -17,6 +17,7 @@ MZViewBase {
 
     _menuTitle: MZI18n.SettingsDnsSettings
 
+    property bool initialized: false
```

Comment:
no longer need the `initialized` property

Reply:
this is to prevent activeFocusChanged to trigger a save before onCompleted.

Reply:
why is it a problem if a save is made before `onCompleted`?

Reply:
because in onCompleted we call 'reset' that sets the correct values. If we save before that, we end up saving random things.